python/lsst/afw/detection/_footprintContinued.py

```diff
@@ -32,6 +32,41 @@
 from ._detection import Footprint
 
 
+def getFootprintIntersection(
+    footprint1: Footprint,
+    footprint2: Footprint,
+    copyPeaks: bool = True,
+) -> Footprint:
+    """Calculate the intersection of two Footprints.
+
+    Parameters
+    ----------
+    footprint1:
```

Comment:
We are apparently still not on documenteer >= 2.0 in pipelines docs and are supposed to be putting type hints here too, sadly.